velox/docs/functions/spark/string.rst

```diff
@@ -141,6 +141,14 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
         SELECT overlay('Spark SQL', 'tructured', 2, 4); -- "Structured SQL"
         SELECT overlay('Spark SQL', '_', -6, 3); -- "_Sql"
 
+.. spark:function:: repeat(str, n) -> varchar
+
+    Returns the string which repeats the given string value ``n`` times. 
```

Comment:
nit: "string value" could be changed as \``string`` to refer to the input variable.